.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -43,6 +43,12 @@ _Do not alter or remove anything below. The following sections will be managed b
 
 * <!-- One or more bullet points for how to technically resolve the issue. -->
 
+### Test Coverage
+* <!-- One or more bullet points for how to implement automated tests to verify the feature works. -->
```

Comment:
While the AC is specific about the language to use, it uses _feature-specific_ language. We should use consistent  language for each template type as it does now.

That is, in the bug report template, we refer to "the issue being resolved" where in the feature request template, we use language that "the feature works".

```suggestion
* <!-- One or more bullet points for how to implement automated tests to verify the issue is resolved. -->
```

This only applies to the "Test Coverage" heading, as the VRT section wording works for both.

cc: @felixarntz

Reply:
SGTM!

Reply:
I updated the ACs to reflect this slight change 👍